bin/test-lxd-storage-volumes-vm

```diff
@@ -121,25 +123,54 @@ do
 	rm vol1.tar.gz
 	rm vol1-optimized.tar.gz
 
+	echo "==> Import custom ISO volume"
+	tmp_iso_dir="$(mktemp -d)"
+	echo foo > "${tmp_iso_dir}/foo"
+	genisoimage -o vol5.iso "${tmp_iso_dir}"
+	rm -f "${tmp_iso_dir}/foo"
+	echo bar > "${tmp_iso_dir}/bar"
+	genisoimage -o vol6.iso "${tmp_iso_dir}"
+	rm -rf "${tmp_iso_dir}"
+	lxc storage volume import "${poolName}" vol5.iso vol5
+	lxc storage volume import "${poolName}" vol6.iso vol6
+	rm -f vol5.iso vol6.iso
+
 	echo "==> Attach custom block volumes to VM"
 	# Both volumes can be attached at the same time. The VM will have /dev/sdb and /dev/sdc.
 	# It doesn't matter which volume is which device as they contain the same content.
 	lxc storage volume attach "${poolName}" vol2 v1
 	lxc storage volume attach "${poolName}" vol3 v1
 
+	echo "==> Attach custom ISO volumes to VM"
+	lxc storage volume attach "${poolName}" vol5 v1
+	lxc storage volume attach "${poolName}" vol6 v1
+	lxc storage volume attach "${poolName}" vol5 v2
+	lxc storage volume attach "${poolName}" vol6 v2
```

Comment:
That tests multi-attach which is good. Since the ISO volume allows concurrent readonly mounts, would it also make sense to test that out by `mount`'ing the same volume(s) in both `v1` and `v2` and then maybe `umount` in one and make sure it still works in the other. Presumably this `mount`/`umount` dance will be more relevant when containers will be supported but wouldn't hurt to test it on VMs too.

Reply:
I changed it to:
```shell
lxc storage volume detach "${poolName}" vol5 v1
lxc exec v2 -- /bin/sh -c 'mount /dev/sr0 /mnt && [ $(cat /mnt/foo) = foo ] && ! touch /mnt/foo && umount /mnt'
```

Reply:
If I followed correctly, the `touch` makes sure the FS is mounted readonly so that's good.

Sorry, I was not clear with what I meant by "concurrent mounts"; let me try with code:

```
# concurrent readonly ISO mounts
# shellcheck disable=2016
lxc exec v1 -- /bin/sh -c 'mount /dev/sr0 /mnt && [ $(cat /mnt/foo) = foo ] && ! touch /mnt/foo'
# shellcheck disable=2016
lxc exec v2 -- /bin/sh -c 'mount /dev/sr0 /mnt && [ $(cat /mnt/foo) = foo ] && ! touch /mnt/foo'
lxc exec v1 -- umount /mnt
lxc exec v2 -- umount /mnt
```